doc/menu_lua_api.txt

```diff
@@ -17,12 +17,12 @@ Gamedata
 --------
 The "gamedata" table is read when calling core.start(). It should contain:
 {
-	playername     = <name>,
-	password       = <password>,
-	address        = <IP/adress>,
-	port           = <port>,
-	selected_world = <index>, -- 0 for client mode
-	singleplayer   = <true/false>,
+    playername     = <name>,
+    password       = <password>,
+    address        = <IP/adress>,
+    port           = <port>,
+    selected_world = <index>, -- 0 for client mode
+    singleplayer   = <true/false>,
```

Comment:
Is this a pure text file, or a .md file renamed to .txt in the style of lua_api.txt? If the latter, then the formatting is wrong all over the place, and this change is incomplete. See https://notabug.org/pgimeno/Gists/src/minetest-clobberxd---menu_lua_api/menu_lua_api.md

I suggest to either leave the formatting alone, or add a separate cleanup commit or even a new PR to clean it up.